core/src/trezor/ui/layouts/tr/__init__.py

```diff
@@ -1268,6 +1270,8 @@ def confirm_solana_tx(
     async def confirm_ethereum_tx(
         recipient: str,
         total_amount: str,
+        _account: str,
```

Comment:
we should probably make the types the same for `TT` and `TR` as for `mercury`. I.e. `str | None`

Reply:
Yes! https://github.com/trezor/trezor-firmware/pull/4175/commits/5b267d4e892ade8cc3908d512a62abb4b25c12b4#diff-b16bb8c99dd53c14619efdcdc4d8bed9b1280510417c880aa08a2644da1451b3R1273